### ISSUE TITLE
checkstyle Eclipse plugin

### ISSUE THREAD

**Issue by angelozerr** [angelozerr/ec4j#52]
@ppalaga now ec4j uses checkstyle, do you know which plugins we must use to foolow the rules (like using tab)

I have found that:

![image](https://user-images.githubusercontent.com/1932211/32200238-e8ffccf0-bdd0-11e7-8358-2e2b72b75c67.png)

How do you support checkstyle in your IDE?

**Comment by ppalaga** [comment]
I have used one of those in the past, my experience was rather bad and I stopped using it then.

Checkstyle will be satisfied, once you (1) clone your default Eclipse formatter, (2) set spaces instead of tabs in it, (3) set removal of trailing white space and (4) set Unix line endings.

As for (3) JDT could not do that well for some time and I started using AnyEdit [1] for that, but JDT was perhaps fixed in between. I kept using AnyEdit because it delivers some other nice features, like various case conversions  

[1] https://marketplace.eclipse.org/content/anyedit-tools
